packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -134,13 +134,7 @@ class LocalFileDestinationProvider implements IDestinationProvider {
 
     if (pipeline && !pipeline.closed) {
       await new Promise<void>((resolve, reject) => {
-        pipeline
-          .on('close', () => {
-            resolve();
-          })
-          .on('error', (e) => {
-            reject(e);
-          });
+        pipeline.on('close', resolve).on('error', resolve);
```

Comment:
Shouldn't we reject instead of resolving in case of an error? Also this is a void promise, but the way it's implemented it would return the error as its resolved value

Reply:
resolve was a typo, but this shouldn't be returning anything, it's got {} around it